src/template.html

```diff
@@ -1,15 +1,19 @@
 <!doctype html>
 <html>
 <head>
-	<meta charset='utf-8'>
-	<meta name='viewport' content='width=device-width,initial-scale=1.0'>
-	<meta name='theme-color' content='#333333'>
+    <meta charset="utf-8">
+	<meta name="viewport" content="width=device-width,initial-scale=1.0">
```

Comment:
This is rather nit picky but you have a mix of tabs and spaces here resulting in misaligned indentation.

Reply:
Also, since I can't comment on the images directly I'll say it here. For general files, it's probably best to keep the permissions as `644` instead of changing to `755`. If you're on Windows you might need to use something like a 'Windows Subsystem for Linux' shell to change the permissions.

Reply:
Hi @MaxMilton i've made those changes for you 👍